requirements_test_pre_commit.txt

```diff
@@ -4,7 +4,7 @@ pre-commit~=3.1;python_version>='3.8'
 bandit==1.7.4
 black==23.1.0
 flake8==6.0.0;python_version>='3.8'
-ruff
+ruff==0.0.254
```

Comment:
I'd rather just have this install latest right? Pinning feels unnecessary as `pre-commit` will get the version as specified anyway.

Reply:
Yeah, I'm fed up with approving dependabot's dozen of MR too.

Reply:
Should I do another commit for that, or change this in another MR?

Reply:
Can we just remove this file perhaps?

Reply:
Let's do a separate commit in this PR

Reply:
We had this argument with Marc, he wanted to have the possibility to use tools without pre-commit, I did not like the duplication personally. I'm not sure about pinging him for that though.

Reply:
With dependabot we are on the latest version anyway in `pre-commit`. Specifying the versions in a file does not improve the workflow significantly as you would still need to `pip install` yourself manually.

Specifying the tools we use seems fine, pinning them to versions and having to update seems a timewaste

Reply:
So let's keep the file and not pin then. Do you think we need another MR of should I just fix this here ?

Reply:
Let's fix here